lib/routes/account.js

```diff
@@ -1046,31 +1043,15 @@ module.exports = function (
 
         function createResponse() {
 
-          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
```

Comment:
@shane-tomlinson @philbooth Per our conversation yesterday, I update `/recovery_email/status` to not reference any sign-in confirmation values. In theory, this endpoint should return the correct value for `verified` regardless if the feature is enabled or not. 

However one thing I noticed while adding this in, since there is no metricsContext, I wouldn't be able to determine if this request is coming from a user that has the sign-in feature enabled and thus return `emailVerified` and `sessionVerified` in the response. The old auth server does not have these additional fields.

cc @rfk